### PR TITLE
Fix issue template support.md

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

Seems like I broke one of our issue templates in #5516 😬 

image

This PR decreses the `about` size to 196 characters

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Support
-about: If you have questions about the Prometheus Operator. If you have questions about helm installation, go to https://github.com/prometheus-community/helm-charts repository. If you have questions about kube-prometheus setup, go to https://github.com/prometheus-operator/kube-prometheus
+about: If you have questions about the Prometheus Operator. Helm? Go to https://github.com/prometheus-community/helm-charts. kube-prometheus ? Go to https://github.com/prometheus-operator/kube-prometheus
```
Comment (xiu) [inline .github/ISSUE_TEMPLATE/support.md:3]:
```suggestion
about: For questions about prometheus-operator. For helm, go to https://github.com/prometheus-community/helm-charts. For kube-prometheus, go to https://github.com/prometheus-operator/kube-prometheus.
```

Comment (xiu) [reply .github/ISSUE_TEMPLATE/support.md:3]: (nit) I'm good either way

**Review by xiu — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/support.md (hunk 1 above)
Comment (JoaoBraveCoding) [reply .github/ISSUE_TEMPLATE/support.md:3]: I like the suggestion just because it reads better but I'm also fine wither way

**Review by JoaoBraveCoding — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/support.md (hunk 1 above)
Comment (xiu) [reply .github/ISSUE_TEMPLATE/support.md:3]: Updated the suggestion :)

**Pull request merged**